koku/api/report/ocp/provider_map.py

```diff
@@ -301,17 +301,18 @@ def __init__(self, provider, report_type):
                             "request": Sum("pod_request_cpu_core_hours"),
                             "limit": Sum("pod_limit_cpu_core_hours"),
                         },
-                        "capacity_aggregate": {
+                        "capacity_dataclass": {
```

Comment:
I don't quite understand this name change. This variable IS an aggregate, not a dataclass.

Reply:
I was just trying to indicate this section of the provider map is used in the dataclasses. I can undo this naming change though.